README.rst

```diff
@@ -20,13 +20,12 @@ What is metatensor-models?
 
 The idea behind ``metatensor-models`` is to have a general hub
 that provide an homogeneous enviroment and user interface to
-train, export, infere ML models and to
-connect those model with various MD engines (e.g. ``LAMMPS``, ``i-PI``, ``ASE`` ...).
-``metatensor-models`` is the tools that tranform every ML architecture in a
+train, export and evaluate ML models and to
+connect those models with various MD engines (e.g. ``LAMMPS``, ``i-PI``, ``ASE`` ...).
+``metatensor-models`` is the tool that transforms every ML architecture in an
```

Comment:
chatgpt failed me

Reply:
?

Reply:
ah sorry it was refered to the commit of @frostedoyster that correct typos in the text generated by chatgpt.
i was not a real comment